### PR TITLE
Minor correction to SupervisedContrastiveLoss

### PR DESCRIPTION
**Pull request by madriss**
Minor correction : in the SupervisedContrastiveLoss class, the logits should use the temperature class attribute instead of the global variable.

### CODE REVIEW

**Comment by madriss** [comment]
@googlebot I signed it!

**Review by fchollet — COMMENTED** [review]
LGTM, thank you for the PR. Please also update the `md` and `ipynb` files.

**Review by fchollet — APPROVED** [review]
